pandas/core/generic.py

```diff
@@ -716,6 +716,64 @@ def swapaxes(self, axis1, axis2, copy=True):
 
         return self._constructor(new_values, *new_axes).__finalize__(self)
 
+    def droplevel(self, level, axis=0):
```

Comment:
anyone think we should add ``inplace`` to make this consistent, with other axis ops (e.g. reset_index / set_index)?

cc @TomAugspurger

Reply:
makes sense to me to expose `inplace`!

Reply:
I'd say leave it out for now, unless someone asks for it.

Reply:
okay, will do!